gradoop-common/src/main/java/org/gradoop/common/storage/impl/hbase/HBaseEPGMStoreFactory.java

```diff
@@ -255,7 +255,7 @@ private static void deleteTablesIfExists(final Configuration config,
    * Deletes a HBase table.
    *
    * @param admin           HBase admin
```

Comment:
You  accidentally replaced "delete" by "remove" everywhere.
Please revert and confirm that only FoodBroker related files are comitted.